vxquery-xtest/test-suites/xqts/ExpectedTestResults/Functions/TraceFunc/fn-trace-10.txt

```diff
@@ -0,0 +1,5 @@
+aaaa
```

Comment:
Do not include files from vxquery-xtest/test-suites/xqts/*. If the user wants to test XQTS, they can follow the instructions on the website to download this tests suite.